assets/js/components/events/EventsDashboard.jsx

```diff
@@ -360,9 +360,11 @@ class EventsDashboard extends Component {
           const integrationResponses = row.integrations && row.integrations.filter(i => i.subCategory === 'uplink_integration_res');
           const integrationError = integrationResponses.findIndex(i => i.status === 'error') === -1 ? false : true;
 
-          const externalIntegrations = row.integrations.filter(i => i.name !== 'Internal Integration');
-          const integrationMissing = externalIntegrations.findIndex(i => i.id === 'no_channel') === -1 ? false : true;
-          
+          const externalIntegrations = row.integrations && row.integrations.filter(i => i.name !== 'Internal Integration');
```

Comment:
do you want to default externalIntegrations to [] here?

otherwise you would have to check for its existence in the next 2 lines

Reply:
good thinking, it worked ok for when the event didn't have integration, but best to be safe.